From 67c2384e444615af6c813a55b0c0a0776e7d1bd6 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Thu, 2 Aug 2012 15:59:35 +0800
Subject: [PATCH 1853/2463] ENGR00219184 HDMI AUDIO: set HDMI audio's SDMA
 priority to high

HDMI use SDMA for buffer switch. Set HDMI audio's SDMA priority to the
same level as other audio device. This adjust can avoid noise due to
SDMA cpu race failing.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 sound/soc/imx/imx-hdmi-dma.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sound/soc/imx/imx-hdmi-dma.c b/sound/soc/imx/imx-hdmi-dma.c
index a743d10..443ff10 100644
--- a/sound/soc/imx/imx-hdmi-dma.c
+++ b/sound/soc/imx/imx-hdmi-dma.c
@@ -938,7 +938,7 @@ static int hdmi_sdma_alloc(struct imx_hdmi_dma_runtime_data *params)
 	dma_cap_mask_t mask;
 
 	params->dma_data.peripheral_type = IMX_DMATYPE_HDMI;
-	params->dma_data.priority = DMA_PRIO_MEDIUM;
+	params->dma_data.priority = DMA_PRIO_HIGH;
 	params->dma_data.dma_request = MX6Q_DMA_REQ_EXT_DMA_REQ_0;
 	params->dma_data.private = &params->sdma_params;
 
-- 
1.7.7.4

